galgebra/mv.py

```diff
@@ -583,10 +583,7 @@ def __getitem__(self, key):
         '''
         return self.grade(key)
 
-    def Mv_str(self):
-        # todo: make this an argument?
-        print_obj = printer.GaPrinter()
-
+    def Mv_str(self, print_obj):
```

Comment:
Hindsight reveals this was a better idea than expected, adding this argument makes this match the signature of the `_sympystr` function that sympy was expecting us to define (xref #273)